Tools/CISupport/build-webkit-org/config.json

```diff
@@ -299,31 +299,31 @@
                   {
                     "name": "Apple-iOS-16-Simulator-Release-GPUProcess-WK2-Tests", "factory": "TestAllButJSCFactory",
                     "platform": "ios-simulator-16", "configuration": "release", "architectures": ["x86_64", "arm64"], "device_model": "iphone",
-                    "additionalArguments": ["--no-retry-failures", "--use-gpu-process", "--child-process=5"],
+                    "additionalArguments": ["--no-retry-failures", "--use-gpu-process", "--child-process=6"],
```

Comment:
Could you please spell the option fully (`--child-processes`)?

Reply:
certainly